File: ios/ObjectiveC/Beacon/1-QuickStart/Starter/QuickComplete/ViewController.m
//
//  ViewController.m
//  QuickStart
//
//  Created by sarra srairi on 22/03/2016.
//  Copyright © 2016 sarra srairi. All rights reserved.
//

#import "ViewController.h"


@interface ViewController ()

@end

@implementation ViewController {
    NSString *feedStatusString;
    AdtagBeaconManager *beaconManager;
    AdtagInitializer *adtagInitializer;
}

@synthesize hashValue;

- (void)viewDidLoad {
    [super viewDidLoad];
    beaconManager = [AdtagBeaconManager shared];
    adtagInitializer = [AdtagInitializer shared];

    // Do any additional setup after loading the view, typically from a nib.
}

- (void) viewDidAppear:(BOOL)animated{
    // Register Range Delegate protocol to your view
    [beaconManager registerInProximityInForeground:self];
    [adtagInitializer registerProximityErrorDelegate:self];
    [super viewDidAppear:animated];
}

- (void) viewDidDisappear:(BOOL)animated{
    [super viewDidDisappear:animated];
    [beaconManager unregisterInProximityInForeground:self];
    [adtagInitializer unregisterProximityErrorDelegate:self];
    //[beaconManager registerAdtagRangeDelegate:nil];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void) onProximityErrorWithErrorType:(NSInteger)errorType message:(NSString *)message {
    _txt_nbrBeacon.text =[NSString stringWithFormat:NSLocalizedString(@"error", @""), errorType, message];
}

-(void) proximityContentsInForegroundWithContents:(NSArray<id<PlaceInAppAction>> *)contents {
    _txt_nbrBeacon.text =[NSString stringWithFormat:NSLocalizedString(@"beaconAround", @""), contents.count];
}



@end

File: ios/SWIFT/Beacon/10-Range/Starter/beaconRange/ViewController.swift
//
//  ViewController.swift
//  beaconRange
//
//  Created by sarra srairi on 10/08/2016.
//  Copyright © 2016 R&D connecthings. All rights reserved.
//

import UIKit
import ATAnalytics
import ATLocationBeacon

class ViewController: UIViewController {
    
    @IBOutlet weak var txt_nbrBeacon: UILabel!
     var beaconManager: ATBeaconManager!
    
    override func viewDidLoad() {
        super.viewDidLoad()
  
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
 
}

